zrml/prediction-markets/src/mock.rs

```diff
@@ -83,6 +83,7 @@ construct_runtime!(
         Balances: pallet_balances::{Call, Config<T>, Event<T>, Pallet, Storage},
         Court: zrml_court::{Event<T>, Pallet, Storage},
         AssetManager: orml_currencies::{Call, Pallet, Storage},
+        ZrmlCurrencies: zrml_currencies::{Call, Pallet, Storage, Event<T>},
```

Comment:
```suggestion
        ZrmlCurrencies: zrml_currencies::{Pallet, Storage},
```

There are no dispatchables or events in the pallet.